lib/medium-admin.php

```diff
@@ -767,7 +767,7 @@ public static function cross_post($post, $medium_post, $medium_user) {
       "canonicalUrl" => $permalink,
       "license" => $medium_post->license,
       "publishStatus" => $medium_post->status,
-      "publishedAt" => mysql2date('c', $post->post_date_gmt),
+      "publishedAt" => mysql2date('c', isset($post->post_date_gmt) ? $post->post_date_gmt : $post->post_date),
```

Comment:
Good question! We have not encountered problems with `post_date` missing or null yet. In the usual case, the post_date should always be set.